### DISCUSSION TITLE
windows11开启剪贴板同步 移动文件夹时提示文件夹正在使用

### DISCUSSION THREAD

**Discussion by deadblue0910** [Jeric-X/SyncClipboard#89] [Q&A]
碰到个奇怪的问题，开启剪贴板同步后，部分文件夹剪切粘贴到其他位置会提示文件夹正在使用，需要点击一次重试才能粘贴。如果直接使用鼠标拖动文件夹则无弹窗。关闭剪贴板同步问题消失

https://github.com/Jeric-X/SyncClipboard/assets/48860218/333303f4-f5af-4dde-a985-1852e014f009

**Comment by Jeric-X** [comment]
很有可能是问题，回头看看

**Comment by Jeric-X** [comment]
如果文件夹很大或者磁盘有点慢，粘贴的操作又很快，是会占用，因为要计算文件夹整体的hash，这个没法避免。你觉得怎么做比较好，遇到剪切的时候不上传从而避免占用？

**Reply by deadblue0910** [reply]
大佬，我不懂程序啊。如果只需要上传文字一样要计算文件夹整体吗？

**Comment by Jeric-X** [comment]
你剪切的文件夹当前就是在上传文件夹呀，也不止是计算时占用，上传的过程中也会占用，这个问题只能通过不上传剪切的文件来彻底解决
